src/App.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable import/extensions */
+import React from 'react';
+import { Routes, Route } from 'react-router-dom';
+import './App.css';
+import Navbar from './Component/navibar';
+import Pagecategory from './pages/PageCatego';
+import BooksPage from './pages/PageBook.js';
+import NoMatch from './pages/nomatch.js';
```

Comment:
Nice job fixing the imports suggested by the previous reviewer 👍. Still, there are some missing here we don't have the `nomatch.js` file inside the `pages` folder and the app is not running correctly kindly make sure you import the correct file.